pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-snmp.md

```diff
@@ -2,48 +2,677 @@
 
-### Create a host using the appropriate template
+</TabItem>
+<TabItem value="Cpu" label="Cpu">
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+| Option                   | Description                                                    | Type |
+|:-------------------------|:---------------------------------------------------------------|:-----|
+| --warning-* --critical-* | Thresholds. Can be: 'managementplane' (%), 'dataplane' (%).    | Mode |
 
-| Field                                | Value                             |
-| :----------------------------------- | :-------------------------------- |
-| Host name                            | *Name of the host*                |
-| Alias                                | *Host description*                |
-| IP                                   | *Host IP Address*                 |
-| Monitored from                       | *Monitoring Poller to use*        |
-| Host Multiple Templates              | Net-PaloAlto-Standard-SNMP-custom |
+</TabItem>
+<TabItem value="Global-Protect-Tunnels" label="Global-Protect-Tunnels">
 
-Click on the *Save* button.
+| Option                   | Description                                                                             | Type |
+|:-------------------------|:----------------------------------------------------------------------------------------|:-----|
+| --warning-* --critical-* | Thresholds. Can be: 'tunnels-usage', 'tunnels-usage-free', 'tunnels-usage-prct' (%).    | Mode |
+
+</TabItem>
+<TabItem value="Hardware" label="Hardware">
+
+| Option               | Description                                                                                                                                                                                      | Type |
+|:---------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----|
+| --component          | Which component to check (Default: '.*'). Can be: 'device'.                                                                                                                                      | Mode |
+| --filter             | Exclude some parts (comma seperated list) Can also exclude specific instance: --filter=device,network.*                                                                                          | Mode |
+| --no-component       | Return an error if no compenents are checked. If total (with skipped) is 0. (Default: 'critical' returns).                                                                                       | Mode |
+| --threshold-overload | Set to overload default threshold values (syntax: section,\[instance,\]status,regexp) It used before default thresholds (order stays). Example: --threshold-overload='device.network,OK,down'    | Mode |
```

Comment:
```suggestion
| --threshold-overload | Use this option to overload the default threshold values (syntax: section,\[instance,\]status,regexp). It can be used before default thresholds (order stays). Example: --threshold-overload='device.network,OK,down'    | Mode |
```